src/Shared/PackageActions/NoOpPackageActions.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+
+namespace Microsoft.CST.OpenSource.PackageActions;
+
+using Contracts;
+using PackageUrl;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class NoOpPackageActions : IManagerPackageActions<IManagerPackageVersionMetadata>
```

Comment:
Is this really no-op if it returns not implemented? I think either the operations should just return empty results or it should be renamed NotImplementedActions.

Reply:
This is a half measure until a larger refactor of ProjectManagers is done with metadata and whatnot, but I agree. I'll just have them return empty results instead of not implemented. That was just the default return when I generated it.